baybe/utils/chemistry.py

```diff
@@ -149,10 +149,15 @@ def smiles_to_fingerprint_features(
     )
     name = f"{encoding.name}_"
     prefix = prefix + "_" if prefix else ""
-    col_names = [
-        prefix + name + f.split("fingerprint")[1]
-        for f in fingerprint_encoder.get_feature_names_out()
-    ]
+    if all("fingerprint" in f for f in fingerprint_encoder.get_feature_names_out()):
```

Comment:
seems functional but please simplify this ugly block a la
```python
no_descriptor_names = all("fingerprint" in f for f in fingerprint_encoder.get_feature_names_out()):
col_names = [prefix + name + (f.split("fingerprint")[1] if no_descriptor_names else f) for f in fingerprint_encoder.get_feature_names_out()]
```
or similar

Reply:
I changed it by adapting your idea and created an explicit `suffixes` list to make the names a bit clearer. Your solution is probably a bit faster, but I liked the format of having explicit `prefix`, `name` and `suffix`. Please resolve if this is now good, otherwise I will use your solution directly.